html/utilities/validation/isValidMonetary.js

```diff
@@ -1,4 +1,4 @@
-function isValidMonetary(value) {
+const isValidMonetary = (value) => {
   const expression = /(^\$?(\d+|\d{1,3}(,\d{3})*)(\.\d+)?$)|^$/;
   return expression.test(value);
 }
```

Comment:
Thanks for the PR! Could you add the `;` at the end of this arrow function please? Thanks again!

Reply:
Sure thing! 😊